Neutaro/chain.json

```diff
@@ -0,0 +1,109 @@
+﻿{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Neutaro",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://neutaro.com/",
+  "pretty_name": "Neutaro",
+  "chain_id": "Neutaro-1",
+  "bech32_prefix": "neutaro",

```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Will work on that now.